app/src/main/java/me/thuanc177/kotsune/libs/animeProvider/allanime/AllAnimeAPI.kt

```
@@ -0,0 +1,554 @@
+
+        // Get available episodes detail if present
+        if (show.has("availableEpisodesDetail")) {
+            val episodesDetail = show.getJSONObject("availableEpisodesDetail")
+            if (episodesDetail.has("sub")) {
+                try {
+                    when (val subValue = episodesDetail.get("sub")) {
+                        is Int -> subValue
+                        is JSONArray -> subValue.length()
+                        else -> null
+                    }
+                } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)